TthCrstl/config.ini

```diff
@@ -1 +1,34 @@
-humStart=260
\ No newline at end of file
+# TthCrstl config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+
+humStart=260
+
+volHum=15
+
+volEff=16
```

Comment:
Are all of these set to the default value?
Might be worth pointing it out which ones are not.

Reply:
All fixed.